ebu_tt_live/scripts/ebu_run.py

```diff
@@ -0,0 +1,47 @@
+"""
+The ``ebu-run`` script is a universal runner for any component or complex interconnected sets of components
+based on a compliant configuration file or set of valid options. The configuration framework uses
+the mozilla/configman package that picks up configuration from command line/config file.
+
+Basic usage:
+------------
+
+  ::
+
+    ebu-run --admin.conf=ebu_tt_live/examples/config/simple-producer.json
+
```

Comment:
If `ebu-run` is a configurator for the entire environment and we pass it a json that includes settings for multiple components, why is this example only for `simple-producer`?